### PR TITLE
Make button component production ready

### PR DESCRIPTION
**Pull request by joshuahughes**
Changes in this branch:-
- tightens button padding
- fixes various colour issues
- fixes a handful of indentation issues picked up by scsslint

### CODE REVIEW

**Comment by paddyduke** [comment]
Why is the button text blue on the website? I can't figure out why an element selector,`a`, is taking precedence over a class, `.Button--primary`.

File: assets/scss/global/components/_button.scss [hunk 1]
```diff
@@ -280,8 +280,9 @@ button:focus {
   display: block;
 }
 
-.Button-label--primary {
-  @include font-size(large);
```
Comment (paddyduke) [inline assets/scss/global/components/_button.scss]: Is this definitely no longer needed? It feels weird that there's a "variant" of this (below), but no base style.

Comment (joshuahughes) [inline assets/scss/global/components/_button.scss]: To create the multi-line buttons, we're just adding in the smaller-sized secondary line above the standard button copy. This requires two spans, each set to `display: block;`. The main button-style stays the same as the base style, so we can just use three classes... .Button-label
.Button-label--secondary
.Button--primary .Button-label--secondary

Comment (paddyduke) [inline assets/scss/global/components/_button.scss]: Cool, makes sense.

**Comment by paddyduke** [comment]
Looks good to me. Pass. 👍

**Pull request merged**